cmd/snap-seccomp/versioninfo.go

```diff
@@ -65,3 +66,12 @@ func showVersionInfo() error {
 	fmt.Fprintln(os.Stdout, vi)
 	return nil
 }
+
+func GoSeccompFeatures() string {
```

Comment:
(nitpick) AFAICT we don't need to export this as it is only needed in tests, something like:
```
--- a/cmd/snap-seccomp/export_test.go
+++ b/cmd/snap-seccomp/export_test.go
@@ -20,9 +20,10 @@
 package main
 
 var (
-       Compile         = compile
-       SeccompResolver = seccompResolver
-       VersionInfo     = versionInfo
+       Compile           = compile
+       SeccompResolver   = seccompResolver
+       VersionInfo       = versionInfo
+       GoSeccompFeatures = goSeccompFeatures
 )
 
 func MockArchUbuntuArchitecture(f func() string) (restore func()) {
diff --git a/cmd/snap-seccomp/versioninfo.go b/cmd/snap-seccomp/versioninfo.go
index 99378bcbba..600bb80937 100644
--- a/cmd/snap-seccomp/versioninfo.go
+++ b/cmd/snap-seccomp/versioninfo.go
@@ -53,7 +53,7 @@ func versionInfo() (string, error) {
        }
 
        major, minor, micro := seccomp.GetLibraryVersion()
-       features := GoSeccompFeatures()
+       features := goSeccompFeatures()
 
        return fmt.Sprintf("%s %d.%d.%d %x %s", myBuildID, major, minor, micro, sh.Sum(nil), features), nil
 }
@@ -67,7 +67,7 @@ func showVersionInfo() error {
        return nil
 }
 
-func GoSeccompFeatures() string {
+func goSeccompFeatures() string {
        features := "-"
        // as more features are added, make this colon-separated
        if actLogSupported() {
```
should work, yes?